<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="banner_x_max">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color: #ADD8E6;
        font: 18pt;
        color: #808080;
        }</string>
   </property>
   <property name="text">
    <string>max</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="banner_y">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color: #ADD8E6;
        font: 18pt;
        color: #808080;
        }</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="expression">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.AppleSystemUIFont</family>
     <pointsize>20</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
        font: 20pt &quot;.AppleSystemUIFont&quot;;
        color: #808080;
        border-style: solid;
        border-width: 1px;
        border-radius: 10%;
        border-color: #808080;
        }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="banner_x">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color: #ADD8E6;
        font: 18pt;
        color: #808080;
        }</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="banner_x_min">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color: #ADD8E6;
        font: 18pt;
        color: #808080;
        }</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="banner_y_min">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color: #ADD8E6;
        font: 18pt;
        color: #808080;
        }</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_min_y">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
        background-color: #F5F5F5;
        font: 16pt &quot;.AppleSystemUIFont&quot;;
        color: #000000;
        }
       </string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>-1000000</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>-5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_max_y">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>500</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
        background-color: #F5F5F5;
        font: 16pt &quot;.AppleSystemUIFont&quot;;
        color: #000000;
        }</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>-1000000</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_max_x">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
        background-color: #F5F5F5;
        font: 16pt &quot;.AppleSystemUIFont&quot;;
        color: #000000;
        }</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>-1000000</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="banner_y_max">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
        background-color: #ADD8E6;
        font: 18pt;
        color: #808080;
        }</string>
   </property>
   <property name="text">
    <string>max</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_box_min_x">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox{
        background-color: #F5F5F5;
        font: 16pt &quot;.AppleSystemUIFont&quot;;
        color: #000000;
        }
       </string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="keyboardTracking">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>-1000000</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>-10</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
